resource_customizations/apps/Deployment/actions/action_test.yaml

```diff
@@ -7,4 +7,9 @@ actionTests:
   expectedOutputPath: testdata/deployment-pause.yaml
 - action: resume
   inputPath: testdata/deployment-pause.yaml
-  expectedOutputPath: testdata/deployment-resume.yaml
\ No newline at end of file
+  expectedOutputPath: testdata/deployment-resume.yaml
+- action: scale
+  inputPath: testdata/deployment.yaml
+  expectedOutputPath: testdata/deployment-scaled.yaml
+  parameters:
+    scale: "6"
```

Comment:
```suggestion
    scale: "6"
```
Don't add missing newline at eof

Reply:
Right, that's effectively a warning. Don't do that. Add a newline.

Reply:
Thank you! should I add this as a comment in the top?

Reply:
No, just add an extra newline at the end of each file

Reply:
Added